chapters/1-reproducibility.tex

```diff
@@ -331,6 +345,11 @@ \subsection{Cataloging and archiving data}
 with directly identifying data and questions of data ownership
 before uploading raw data to any third-party server, whether public or not;\index{data ownership}
 this is a legal question for your home organization.
+If essential data must be placed under restricted use or restricted access,
+develop a plan for storing that portion of the data
+separately from publishable information,
+as well as a protocol for allowing access in the future
+for replicators or reviewers under appropriate access agreements.
```

Comment:
is this the response to the comment about archiving data? i think it could be a little clearer, like it's not necessarily going to make sense to separately save the identified data if that is an integral part of the analysis dataset. also, 'essential' here is confusing, i assume that you mean data required for the analysis, and it would be clearer to simply say that. 
this applies equally to data that cannot be published because it has PII and data that cannot be published based on the ownership terms, should be more explicit about this in the paragraph.

Reply:
Additionally, i do not think we have sufficiently addressed comment 3.1.4, how to maximize transparency and reproducibility within the constraints of restricted use data that cannot be published for legal reasons or research designs with PII necessary for analysis. we say in the matrix, "Our recommendation is that all the data that can be published be published, and when data cannot be published, researchers should try to publish as much metadata as allowed, including information on how the data was obtained, what fields it contains and aggregations or descriptive statistics. Even if the data cannot be published, it is very rare for codes to contain restricted information or be barred from publication, so the code should still be made available. Finally, we recommend that reproducibility efforts be considered when designing the IRB and data licensing agreement, so there are specific conditions (such as a secure server or cold room) under which representatives from journals may access the data for the sole purpose of independently verifying results.", we should add all that (not sure that here is the exact right spot but within this section)

Reply:
```suggestion
If data that is required for analysis must be placed under restricted use or restricted access,
including data that can never be distributed directly by you to third parties,
develop a plan for storing that data separately from publishable information.
This will allow you to maximize transparency by having a clear release package
as well as providing instructions or developing a protocol for allowing access in the future
for replicators or reviewers under appropriate access agreements.
```

Revised for first comment, going to insert the second comment language elsewhere

Reply:
Done in 570dd009575622c36ef17c65abfa5feaee0a3c1d